doc/attributes_mathMap.dox

```diff
@@ -2,7 +2,7 @@ namespace ast {
 /**
 @page mathmap_attributes MathMap Attributes
 
-@section MathMap_Attributes @ref MathMap Attributes
+@section MathMap_Attributes MathMap Attributes
```

Comment:
Took me a while to realize that this is a second header on the page itself. It looks a little odd, but I guess that's out of scope for this ticket.